pytensor/tensor/slinalg.py

```diff
@@ -44,19 +44,20 @@ class Cholesky(Op):
         nans instead.
     """
 
-    # TODO: inplace
     # TODO: for specific dtypes
     # TODO: LAPACK wrapper with in-place behavior, for solve also
 
     __props__ = ("lower", "destructive", "on_error")
     gufunc_signature = "(m,m)->(m,m)"
 
-    def __init__(self, *, lower=True, on_error="raise"):
+    def __init__(self, *, lower=True, on_error="raise", overwrite_a=False):
         self.lower = lower
-        self.destructive = False
+        self.destructive = overwrite_a
```

Comment:
My clever refactored function only works if the init argument has the same name as the property